tests/end_to_end/remoteTestData/reads/remoteTest/urls.json

```diff
@@ -0,0 +1,5 @@
+{
+    "urls": [
+        "http://localhost:8002/bam.bam"
```

Comment:
Why don't we reuse the existing files in `tests/data` rather than adding in new data just for this test?